### DISCUSSION TITLE
ERROR    Status code 400

### DISCUSSION THREAD

**Discussion by jacobjahnke** [NREL/REopt-API-Analysis#66] [1 - General REopt Questions]
Hello, I am running the single_scenario_example.impyb and when I run the api_response I get the following error (clipped):
`main         ERROR    Status code 400.`

I also see: 

`Error. No optimization task has been created. See messages for more information. Note that inputs have default values filled in.`

I am just running the code as is, what am I missing? Thanks!

**Comment by adfarth** [comment]
Could you try running it again and relay if you still encounter the error?

**Reply by jacobjahnke** [reply]
I tried again and am still getting the same error.

**Reply by adfarth** [reply]
Interesting, running the notebook as-is is currently working as expected for me. Are you using the latest version (pulled all changes)? Have you modified the `post_1` prior to running? Are you able to share the `run_uuid`?

**Reply by jacobjahnke** [reply]
Hmm, I had just forked the latest version this morning. I tried opening again ran and all worked! So I am not sure what the fix was but I got it now!

**Reply by jacobjahnke** [reply]
Hello, I have changed some of the scenario inputs now and am getting the error above. What does the error mean?

**Reply by adfarth** [reply]
The 400 error indicates that at least one of the inputs is invalid. You can see the min and max allowable values for each input in `nested_inputs.py` [here](https://github.com/NREL/REopt_API/blob/master/reo/nested_inputs.py). In the returned error message in the Jupyter Notebook, you can also find the key that is invalid e.g.:
`"messages": {"error": "Invalid inputs. See \'input_errors\'.", "input_errors": ["analysis_years value (-20) in Scenario>Site>Financial is less than the allowable min 1"], ` (you have to dig through the message a bit to get to this).

**Reply by jacobjahnke** [reply]
Ahh I see, thank you so much!
